pkg/ibmcsidriver/node.go

```diff
@@ -302,11 +305,15 @@ func (csiNS *CSINodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeSt
 
 	// FormatAndMount will format only if needed
 	ctxLogger.Info("Formating and mounting ", zap.String("source", source), zap.String("stagingTargetPath", stagingTargetPath), zap.String("fsType", fsType), zap.Reflect("options", options))
-	err = csiNS.Mounter.NewSafeFormatAndMount().FormatAndMount(source, stagingTargetPath, fsType, options)
+	err = csiNS.SafeFormatAndMount.FormatAndMount(source, stagingTargetPath, fsType, options)
 	if err != nil {
 		return nil, commonError.GetCSIError(ctxLogger, commonError.FormatAndMountFailed, requestID, err, source, stagingTargetPath)
 	}
 
+	if _, err := mountmgr.Resize(csiNS.SafeFormatAndMount, devicePath, stagingTargetPath); err != nil {
```

Comment:
This will not be change if we implement new changes in https://github.com/IBM/ibm-csi-common lib

Reply:
Surely the logic can move to this lib. But it will require quite a few changes to the current lib code as well to allow command faking. Basically what I tried to do here was to avoid calling this constructor here (`NewSafeFormatAndMount()`) and call it only once driver is being constructed because that's where the fakes get in.

So if I understand it well to keep this behavior in the lib then  [this](https://github.com/IBM/ibm-csi-common/blob/master/pkg/mountmanager/safe-mounter.go#L31) would have to move [here](https://github.com/IBM/ibm-csi-common/blob/fe384106592ad5b5586aff5dfdc1c89fd52a4ee3/pkg/mountmanager/safe-mounter.go#L44).

And then ofcourse it would make sense to move the "custom faker" there as well as I [wrote before](https://github.com/kubernetes-sigs/ibm-vpc-block-csi-driver/pull/100/files/528af8273e1c45fdd5f515340fff36497dd0dd67#r1001816301).

Reply:
sure @RomanBednar , actually as of now we have two safeFormatAndMount objectand only one of them is useful, that one reason another let the code be in previous format. I asked @ambiknai to check the possibility to move this code to lib for me it can we done.

Reply:
@RomanBednar  Sorry it took some time to get back on this PR. ? Would it be  possible for you to contribute  above discussed changes in [IBM/ibm-csi-common ](https://github.com/IBM/ibm-csi-common) repo

Reply:
FYI, test-infra PR for sanity is merged .

Reply:
@ambiknai Sure, I'll take a look.